roscala/src/test/scala/coop/rchain/rosette/prim/TupleSpec.scala

```diff
@@ -22,6 +22,15 @@ class TupleSpec extends FlatSpec with Matchers {
     monitor = null
   )
 
+  /** null? */
+  "obNullQ" should "return true when first argument is tuple.NIL, otherwise return false" in {
+    val newCtxt = ctxt.copy(nargs = 1, argvec = Tuple(Tuple.NIL))
+    obNullQ.fnSimple(newCtxt) should be(Right(RblBool(true)))
+
+    val newCtxt2 = ctxt.copy(nargs = 1, argvec = Tuple(Tuple(Fixnum(1))))
+    obNullQ.fnSimple(newCtxt2) should be(Right(RblBool(false)))
+  }
```

Comment:
Please add a test case when `argvec` does not contain an argument.

Reply:
When using the `checkArgumentMismatch` annotation, there is no need to create the test, am I right?

Reply:
Yeah, I think so.